config/rbac/base/kustomization.yaml

```diff
@@ -1,18 +1,10 @@
-# Adds namespace to all resources.
-namespace: gatekeeper-system
-
-# Value of this field is prepended to the
-# names of all resources, e.g. a deployment named
-# "wordpress" becomes "alices-wordpress".
-# Note that it should also match with the prefix (text before '-') of the namespace
-# field above.
-namePrefix: gatekeeper-operator-
```

Comment:
We need this so that the resources' names in this directory get prefixed with the `gatekeeper-operator-` name.